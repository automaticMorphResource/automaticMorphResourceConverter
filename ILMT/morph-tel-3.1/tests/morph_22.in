1	ivi	unk
1	annI	unk	
1	vAriki	unk	
1	prakqwi	unk	
1	prasAxiMcina	unk	
1	varame	unk	
1	.	unk	
