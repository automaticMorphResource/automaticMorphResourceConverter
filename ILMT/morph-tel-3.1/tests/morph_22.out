1	ivi	unk	<fs af='ivi,pn,,sg,,d,0,0' deri_cat='n'>|<fs af='ivi,pn,,sg,,o,ti,ti' deri_cat='n'>|<fs af='ixi,pn,fn,pl,3,d,0,0'>
2	annI	unk	<fs af='annI,avy,,,,,0,0_avy'>|<fs af='anni,pn,,pl,,,0,0_V'>
3	vAriki	unk	<fs af='vAriki,n,,sg,,d,0,0'>|<fs af='vAriki,n,,sg,,o,ti,ti'>|<fs af='vAru,pn,fm,pl,3,,ki,ki'>
4	prakqwi	unk	<fs af='prakqwi,n,,sg,,d,0,0'>|<fs af='prakqwi,n,,sg,,o,ti,ti'>
5	prasAxiMcina	unk	<fs af='prasAxiMcu,v,any,any,any,,ina,ina'>
6	varame	unk	<fs af='varaM,n,,sg,,,0,0_e'>
7	.	unk	<fs af='&dot;,punc,,,,,,'>
